opct/chain.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "opct",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "OPCT Chain",
+  "chain_id": "opct_1",
+  "bech32_prefix": "opct",
```

Comment:
Registration of this prefix here is recommended: https://github.com/satoshilabs/slips/blob/master/slip-0173.md